apiserver/utility/utility.go

```diff
@@ -44,12 +45,15 @@ func ValidateBitriseConfigAndSecret(bitriseConfig, secretsConfig string) (*Warni
 		return nil, fmt.Errorf("Validation failed: %s", strings.Join(errorStrs, " | "))
 	}
 
-	warningItems := &WarningItems{Config: []string{}, Secrets: []string{}}
+	warningItems := WarningItems{}
 	if len(bitriseWarns) > 0 {
 		warningItems.Config = bitriseWarns
 	}
```

Comment:
can't we do? `if len(bitriseWarns) > 0 {} else { return nil, nil }` instead of using `reflect`?